manager/testdata/blueprint-read.yaml

```diff
@@ -12,32 +12,34 @@ spec:
   selector:
     matchLabels:
       app: notebook
-  entrypoint: notebook
-  flow:
-    name: notebook
-    steps:
-    - arguments:
-        read:
-          - source:
-              connection:
-                name: S3
-                s3:
-                  bucket: fybrik-test-bucket
-                  endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
-                  object_key: small.kp.encrypted.parq
-                type: 2
-              vault:
-                address: http://vault.fybrik-system:8200
-                authPath: /v1/auth/kubernetes/login
-                role: module
-                secretPath: "/v1/kubernetes-secrets/secret-name?namespace=default"
-              format: parquet
-            assetID: xyz
-      name: notebook-read-module
-      template: read-module
-  templates:
-  - name: read-module
-    kind: FybrikModule
-    flow: read
+  cluster: cluster1
+  modules:
+  - name: notebook-read-module
+    instanceName: notebook-read-module-instance1
     chart:
       name: ghcr.io/fybrik/fybrik-template:0.1.0
+    apis:
```

Comment:
Should the api be defined in a blueprint? I don't see it defined in testdata/blueprint.yaml, in generation or usage of blueprints.

Reply:
Agreed there's no need for api in blueprint.  Removed it.